_config.yaml

```diff
@@ -34,11 +34,13 @@ header_pages:
   # Note that when first cloned, the `workshop` directory will not exist,
   # as described at https://github.com/AlexsLemonade/training-specific-template#customizing-the-new-repository-for-an-individual-training-workshop
   - workshop/SCHEDULE.md
-  - workshop/workshop-structure.md
+  - workshop/participant_information.md
   - workshop/workshop-materials.md
+  - workshop/single-cell-resources.md
```

Comment:
I might move this to the end? Not sure... Ordering is hard.

Alternatively, we could just make a link to this doc on the materials page.